src/List/Item.tsx

```diff
@@ -18,10 +19,10 @@ export default class Item extends React.PureComponent {
 
   render() {
     const {cssClass} = Item;
-    const {children, className} = this.props;
+    const {children, className, onClick} = this.props;
 
     return (
-      <li className={classnames(cssClass.CONTAINER, className)}>
+      <li className={classnames(cssClass.CONTAINER, className)} onClick={onClick}>
```

Comment:
for a11y, maybe we should wrap `{children}` in a <button> if an onClick is specified so that the interaction's more standard (pointer shows up, user can tab through items, etc)

and then we should probably also add a hover state - darken the background a bit for that row?

we can move the padding that's currently on `cssClass.CONTAINER` onto another class and assign that to either the <button> or a plain <div> depending on whether or not onClick is specified